ci/run_pytests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+# Support invoking run_pytests.sh outside the script directory
+cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../
+
+export UCX_TLS="${UCX_TLS:-tcp,cuda_copy}"
```

Comment:
What are the environments where this will normally run? We originally set this value because of CI limitations on interconnects (e.g., no InfiniBand/NVLink). If this is going to run on development test environments where this may be available we might want not to set anything. In fact, we could remove this entirely today, `UCX_TLS=all` (default) should work well anywhere on recent UCX versions.

Reply:
Yeah I wasn't sure if this should be set, so I'm being conservative. A user can override it, but I didn't want the tests to fail if they don't.

My main goal of this PR (and all the other similar ones I've filed tonight in other repos) is to provide a test script we can run in the DLFW/NVAIE builds. The test recipes change so frequently, that it's difficult to keep the DLFW tests in sync for all repos.

I typically run these tests on my local workstation, or on a lab DGX if I need another machine. Eventually we want to run the tests in Blossom or Colossus, but I can't predict what will be available in those environments.

Reply:
If you run on DGX lab machines I'd suggest we remove `UCX_TLS` completely and use UCX's default from now on, this applies to regular CI as well. Would you mind removing `UCX_TLS` both here and in `ci/test_python.sh`?